web/matrixdef

```diff
@@ -3382,6 +3636,9 @@ BeginIter det-pc{i} "a Position Class" 1
       MultiSelect value "Determiner Position Class {i} Lexical Rule Type {j} Feature {k} Value" "Value: " ""
       fillvalues p=det-pc{i}_lrt{j}_feat{k}_name
 
+      Select head "Determiner Position Class {i} Lexical Rule Type {j} Feature {k} Head" "Specified on: " "" "" "toggle_element.call(this, 'adj-pc{i}_lrt{j}_adj_pc_note', 'on', ['subj', 'mod'])"
```

Comment:
I think you don't want this `toggle_element.call(this, 'adj-pc{i}_lrt{j}_adj_pc_note', 'on', ['subj', 'mod'])` call here, that's for showing the switching type LRTs message when defining a switching type adjective.

Reply:
Also, I'm not sure why we want a selectable that only has one option at all?

Reply:
Right! We don't. I think what happened is I couldn't, at the time, figure out how to do it differently and decided to simply copy what was working :). I will see if I can quickly figure it out now, and if not will file an issue for the future...

Reply:
OK, thanks for pointing this out! I can't test this at all because the website simply doesn't work with python3 version. I can test it separately in the python2 version but I think it's best to leave it till the time I can properly test it in this version.